keps/sig-node/753-sidecar-containers/README.md

```diff
@@ -1297,18 +1576,26 @@ These goals will help you determine what you need to measure (SLIs) in the next
 question.
 -->
 
+- sidecar init containers are running and restarted as expected
+- jobs with sidecars are completed as expected
+- TGPS is enforced for pods with sidecar containers
+
 ###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
 
 <!--
 Pick one more of these and delete the rest.
 -->
 
-- [ ] Metrics
-  - Metric name:
-  - [Optional] Aggregation method:
-  - Components exposing the metric:
-- [ ] Other (treat as last resort)
-  - Details:
+- [X] Metrics
```

Comment:
Do we have any k8s-native metrics?

kube-state-metrics is not a core component and as long as it makes sense for monitoring individual components, I would really like to see some k8s-native metric that would tell me at least something about sidecars.

Maybe #init_containers started? Maybe number of actions done on init_containers [split by action being a label] or sth?
I don't have any strong opinion, but I really would like to see some kubelet-native metric here.

@SergeyKanzhelev - FYI